subcommand/inject-connect/command_test.go

```diff
@@ -153,11 +153,6 @@ func TestRun_FlagValidation(t *testing.T) {
 			},
 			expErr: "request must be <= limit: -lifecycle-sidecar-cpu-request value of \"50m\" is greater than the -lifecycle-sidecar-cpu-limit value of \"25m\"",
 		},
-		{
```

Comment:
I removed this test because we're using `cfg := api.DefaultConfig()` now defaults the address if it's not set in the env var (and therefore doesn't error out). This no longer will error if that env var isn't set, instead it will set the address to 127.0.0.1 (which will likely break later) so this is a reduction in safety but does match the other locations mentioned in the PR description.